main.cpp

```diff
@@ -532,6 +621,15 @@ void OscillatorSection::setName(std::string newOscName)
     std::cout << "Previous oscillator name " << tempOscName << " has been changed to " << oscName << std::endl;
 }
 
+void OscillatorSection::automatePan(float targetPan)
+{
+    while (pan < targetPan)
+        {
+            pan += 0.1f;
+            std::cout << "Pan is " << pan << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise